content/resources/windows_audit_policy/_index.md

```diff
@@ -0,0 +1,140 @@
+      a System Access Control List to named objects to enable auditing of container
+      objects such as directories.
+- property: audit_base_objects
+  ruby_type: true, false
+  required: false
+  description_list:
+  - markdown: Setting this audit policy option to true will force the system to assign
+      a System Access Control List to named objects to enable auditing of base objects
+      such as mutexes.
+- property: crash_on_audit_fail
+  ruby_type: true, false
+  required: false
+  description_list:
+  - markdown: Setting this audit policy option to true will cause the system to crash
+      if the auditing system is unable to log events.
+- property: exclude_user
+  ruby_type: String
+  required: false
+  description_list:
+  - markdown: The audit policy specified by the category or subcategory is applied
+      per-user if specified. When a user is specified, exclude user. Include and exclude
+      cannot be used at the same time.
+- property: failure
+  ruby_type: true, false
+  required: false
+  description_list:
+  - markdown: Specify failure auditing. By setting this property to true the resource
+      will enable failure for the category or sub category. Success is the default
+      and is applied if neither success nor failure are specified.
+- property: full_privilege_auditing
+  ruby_type: true, false
+  required: false
+  description_list:
+  - markdown: Setting this audit policy option to true will force the audit of all
+      privilege changes except SeAuditPrivilege. Setting this property may cause the
+      logs to fill up more quickly.
+- property: include_user
+  ruby_type: String
+  required: false
+  description_list:
+  - markdown: The audit policy specified by the category or subcategory is applied
+      per-user if specified. When a user is specified, include user. Include and exclude
+      cannot be used at the same time.
+- property: subcategory
+  ruby_type: String, Array
+  required: false
+  description_list:
+  - markdown: The audit policy subcategory, specified by GUID or name. Applied system-wide
+      if no user is specified.
+- property: success
+  ruby_type: true, false
+  required: false
+  description_list:
+  - markdown: Specify success auditing. By setting this property to true the resource
+      will enable success for the category or sub category. Success is the default
+      and is applied if neither success nor failure are specified.
+examples: |
+  **Set Logon and Logoff policy to "Success and Failure"**:
+
+  ```ruby
+  windows_audit_policy "Set Audit Policy for 'Logon and Logoff' actions to 'Success and Failure'" do
+    subcategory   %w(Logon Logoff)
+    success        true
+    failure        true
+    action         :set
+  end
+  ```
+
+  **Set Credential Validation policy to "Success"**:
+
+  ```ruby
+  windows_audit_policy "Set Audit Policy for 'Credential Validation' actions to 'Success'" do
+    subcategory   "Credential Validation"
```

Comment:
Weird that the indentation for this property is off in two places. 🤔

Reply:
Fixed